lib/server.ex

```diff
@@ -86,10 +86,7 @@ defmodule GymTcpApi.Server do
   end
 
   def write_line(socket, {:ok, text}) do
-    if String.strip(text) === "" do
-    else
-      :gen_tcp.send(socket, text)
-    end
+    :gen_tcp.send(socket, text)
```

Comment:
"(FunctionClauseError) no function clause matching in String.Break.trim_leading/1"